stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channel/info/ChannelInfo.kt

```diff
@@ -154,42 +155,55 @@ public fun ChannelInfo(
 
                 Text(
                     modifier = Modifier.padding(horizontal = 16.dp),
-                    text = title,
-                    style = ChatTheme.typography.bodyBold,
+                    text = ChatTheme.channelNameFormatter.formatChannelName(selectedChannel),
+                    style = ChatTheme.typography.title3Bold,
                     color = ChatTheme.colors.textHighEmphasis,
                     maxLines = 1,
                     overflow = TextOverflow.Ellipsis
                 )
                 Text(
-                    text = LocalContext.current.resources.getQuantityString(
-                        R.plurals.stream_compose_channel_members,
-                        channelMembers.size,
-                        channelMembers.size,
-                        onlineMembers
-                    ),
+                    text = subtitle,
                     style = ChatTheme.typography.footnoteBold,
                     color = ChatTheme.colors.textLowEmphasis,
                 )
 
-                LazyRow(
-                    modifier = Modifier
-                        .fillMaxWidth()
-                        .padding(top = 8.dp, bottom = 16.dp)
-                        .align(Alignment.CenterHorizontally),
-                    horizontalArrangement = Arrangement.spacedBy(12.dp),
-                    contentPadding = PaddingValues(start = 16.dp, end = 16.dp)
-                ) {
-                    items(selectedChannel.members) { member ->
-                        ChannelInfoUserItem(member = member)
-                    }
-                }
+                ChannelMembers(otherMembers)
 
                 ChannelOptions(channelOptions, onChannelOptionClick)
```

Comment:
@bychkovdmitry One things I noticed is missing is "muting channels" - should we add this option to our list of channel options?

Reply:
This looks like a something new. I wasn't able to find this option in any SDK. But yes, we should do it. Will do it later in another PR as this looks like is a big change (handle action in viewmodel, reflect the muted state in the UI with different labels "mute channel"/"unmute channel").

Reply:
Cool!

I'll open a ticket for this and we can focus on it in another PR!